django_basecrm/helpers.py

```diff
@@ -180,7 +192,8 @@ def get_stages():
         stages = get_stages_from_api()
     return stages
 
-def get_users():
+
+def get_users(**kwargs):
```

Comment:
This is the change - adding support for `**kwargs` which is a pass-through to `get_users_from_api`.